JS/sideNav.js

```diff
@@ -0,0 +1,10 @@
+const hamburger = document.querySelector("#hamburger")
+const sidenav = document.querySelector("#sidenav")
+
+hamburger.addEventListener("click", () => {
+    if(sidenav.style.width === "12rem"){
+        sidenav.style.width = "0"
+    } else {
+        sidenav.style.width = "12rem"
```

Comment:
It would be better to use addClass / removeClass instead of directly manipulating styles